week-2/implement/vat.js

```diff
@@ -8,3 +8,10 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function Vat(num){
+    const NumWithoutMoneySymbols=num.substring(1,num.length)
```

Comment:
variable names should also start with a lower case letter.

Reply:
What happens if the num is passed without a £ sign?

Reply:
It will remove the first digit of the number instead of sign "£".